en/test_and_ci/integration_testing_mavsdk.md

```diff
@@ -20,7 +20,7 @@ Download the matching .deb file (whichever is correct for your system) from [MAV
 Install it using `dpkg`, e.g.:
 
 ```sh
-sudo dpkg -i mavsdk_0.23.0_ubuntu18.04_amd64.deb
+sudo dpkg -i mavsdk_0.24.0_ubuntu18.04_amd64.deb
```

Comment:
@julianoes I've removed all of these installation instructions and lined to the mavsdk ones in https://github.com/PX4/Devguide/pull/975
Can you merge that and update this appropriately.

If you disagree with the strategy we can discuss further